go.mod

```diff
@@ -13,15 +13,17 @@ require (
 	github.com/go-stack/stack v1.8.0 // indirect
 	github.com/golang/protobuf v1.3.5
 	github.com/google/go-cmp v0.4.0 // indirect
+	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/grpc-ecosystem/grpc-gateway v1.14.3
 	github.com/jonboulle/clockwork v0.1.1-0.20190114141812-62fb9bc030d1
 	github.com/kabukky/httpscerts v0.0.0-20150320125433-617593d7dcb3
 	github.com/nikkolasg/hexjson v0.0.0-20181101101858-78e39397e00c
 	github.com/nikkolasg/slog v0.0.0-20170921200349-3c8d441d7a1e
+	github.com/prometheus/client_golang v1.5.1
 	github.com/soheilhy/cmux v0.1.4
 	github.com/stretchr/testify v1.4.0
 	github.com/urfave/cli/v2 v2.2.0
-	go.dedis.ch/kyber/v3 v3.0.9
+	go.dedis.ch/kyber/v3 v3.0.9 // indirect
```

Comment:
Why is this appearing ...  ? We shouldn't rely in anyway on dedis.ch repo anymore. Any idea ? Maybe a `go mod tidy` would clean it ?

Reply:
`go mod tidy` did the trick!